deepmedic/frontEnd/configParsing/trainSessionParams.py

```diff
@@ -7,346 +7,455 @@
-        print("ERROR: The parameter \"classicMom0OrNesterov1\" must be given 0 or 1. Omit for default. Exiting!"); exit(1)
+    def errorRequireMomentumClass0Nestov1():
+        print("ERROR: The parameter \"classicMom0OrNesterov1\" must be given 0 or 1. Omit for default. Exiting!");
+        exit(1)
+
     @staticmethod
-    def errorRequireMomValueBetween01() :
-        print("ERROR: The parameter \"momentumValue\" must be given between 0.0 and 1.0 Omit for default. Exiting!"); exit(1)
+    def errorRequireMomValueBetween01():
+        print("ERROR: The parameter \"momentumValue\" must be given between 0.0 and 1.0 Omit for default. Exiting!");
+        exit(1)
+
     @staticmethod
-    def errorRequireMomNonNorm0Norm1() :
-        print("ERROR: The parameter \"momNonNorm0orNormalized1\" must be given 0 or 1. Omit for default. Exiting!"); exit(1)
-    
+    def errorRequireMomNonNorm0Norm1():
+        print("ERROR: The parameter \"momNonNorm0orNormalized1\" must be given 0 or 1. Omit for default. Exiting!");
+        exit(1)
+
     def __init__(self,
-                log,
-                mainOutputAbsFolder,
-                folderForSessionCnnModels,
-                folderForPredictionsVal,
-                folderForFeaturesVal,
-                num_classes,
-                model_name,
-                cfg):
-        
-        #Importants for running session.
+                 log,
+                 mainOutputAbsFolder,
+                 folderForSessionCnnModels,
+                 folderForPredictionsVal,
+                 folderForFeaturesVal,
+                 num_classes,
+                 model_name,
+                 cfg):
+
+        # Importants for running session.
         # From Session:
         self.log = log
         self.mainOutputAbsFolder = mainOutputAbsFolder
-        
+
         # From Config:
-        self.sessionName = self.getSessionName( cfg[cfg.SESSION_NAME] )
-        
+        self.sessionName = self.getSessionName(cfg[cfg.SESSION_NAME])
+
         abs_path_to_cfg = cfg.get_abs_path_to_cfg()
-        abs_path_to_saved = getAbsPathEvenIfRelativeIsGiven( cfg[cfg.SAVED_MODEL], abs_path_to_cfg ) if cfg[cfg.SAVED_MODEL] is not None else None # Load pretrained model.
-        self.savedModelFilepath = check_and_adjust_path_to_ckpt( self.log, abs_path_to_saved) if abs_path_to_saved is not None else None
-        
-        #====================TRAINING==========================
+        abs_path_to_saved = getAbsPathEvenIfRelativeIsGiven(cfg[cfg.SAVED_MODEL], abs_path_to_cfg) if cfg[
+                                                                                                          cfg.SAVED_MODEL] is not None else None  # Load pretrained model.
+        self.savedModelFilepath = check_and_adjust_path_to_ckpt(self.log,
+                                                                abs_path_to_saved) if abs_path_to_saved is not None else None
+
+        self.tensorboardLog = bool(cfg[cfg.TENSORBOARD_LOG])
```

Comment:
Value of self.tensorboardLog is not logged at print_params(...).

In general, when we add a new parameter to the train/test/model config files, we also make sure to print it, so that its value is logged. So that in future, we can go back to the log files and make sure that the session was performed with the values we wanted it to.

See Line 458+